pandas/core/ops.py

```diff
@@ -990,6 +991,26 @@ def _construct_divmod_result(left, result, index, name, dtype):
     )
 
 
+def dispatch_to_extension_op(left, right, op_name):
+    """
+    Assume that left is a Series backed by an ExtensionArray,
+    apply the operator defined by op_name.
+    """
+
+    method = getattr(left.values, op_name, None)
+    if method is not None:
+        res_values = method(right)
```

Comment:
Same comment as before, can we do here `op(left, right)` instead of getting the method manually ? (similar as what happens in `dispatch_to_index_op`) ?

Reply:
I've made that change.  Thanks.